packages/@misk/cli/src/utils/generate.ts

```diff
@@ -10,24 +17,41 @@ import {
 } from "./templates"
 import { getVersion } from "./changelog"
 
-export const generateBuildFiles = async () => {
-  const pkg = await fs.readJson(Files.package)
-  const miskTab: IMiskTabJSON = await fs.readJson(Files.miskTab)
+const tag = "generate"
+
+export const generateBuildFiles = async (...args: any) => {
+  const { dir } = parseArgs(...args)
+  logDebug(tag, "", dir)
+  let pkg = {}
+  if (fs.existsSync(path(dir, Files.package))) {
+    pkg = fs.readJson(path(dir, Files.package))
+  }
+  const miskTab: IMiskTabJSON = await fs.readJson(path(dir, Files.miskTab))
   // Write out fresh files
   await [
-    fs.writeJson(Files.tsconfig, createTsconfig(miskTab), JsonOptions),
-    fs.writeJson(Files.tslint, tslint, JsonOptions),
-    fs.writeJson(Files.package, createPackage(miskTab, pkg), JsonOptions),
-    fs.writeFile(Files.gitignore, gitignore),
-    fs.writeFile(Files.webpack, webpack)
+    fs.writeJson(
+      path(dir, Files.tsconfig),
+      createTsconfig(miskTab),
+      JsonOptions
+    ),
+    fs.writeJson(path(dir, Files.tslint), tslint, JsonOptions),
+    fs.writeJson(
+      path(dir, Files.package),
+      createPackage(miskTab, pkg),
+      JsonOptions
+    ),
+    fs.writeFile(path(dir, Files.gitignore), gitignore),
+    fs.writeFile(path(dir, Files.webpack), webpack)
   ]
 
   // Append non-package JSON files with generated by comment
-  await fs.writeFile(Files.tsconfig, generatedByCLI, { flag: "a" })
-  await fs.writeFile(Files.tslint, generatedByCLI, { flag: "a" })
-  console.log(
-    `[GENERATE] Up to date build files generated using Misk Web version ${getVersion(
+  fs.writeFile(path(dir, Files.tsconfig), generatedByCLI, { flag: "a" })
```

Comment:
huh, what's this 'flag: "a"'?

Reply:
`append` mode https://nodejs.org/api/fs.html#fs_file_system_flags